app/views/indicators/_inactive_indicators.html.erb

```diff
@@ -17,7 +17,7 @@
         <tr>
           <td><%= link_to i.description, indicator_path(i) %></td>
           <td><%= link_to i.level.name, level_path(i.level) %></td>

```

Comment:
Remove link to level. (Really just a note for later you don't have to make the change in this PR)